docs/index.html

```diff
@@ -1,65 +1,108 @@
-<div class="index-corner">&nbsp;</div>
+<h1 class="calibre white normal f3 lh-title f2-l">
```

Comment:
This is huge on my mac. I think this should be f3 while the headings on each text box should be f2. What do you think?

Reply:
Yes, I find it huge myself. Happy to be changed. Perhaps, only `f2-l` needs to be removed, since it overrides `f3` for large screens